vta/tutorials/frontend/deploy_classification.py

```diff
@@ -56,9 +56,14 @@
 from tvm.contrib.debugger import debug_runtime
 from tvm.relay import transform
 
+os.chdir(dirname(os.path.abspath(__file__)))
+os.chdir(join(pardir, pardir, pardir))
+sys.path.append(os.getcwd())
```

Comment:
CI is complaining about the usage of `__file__` here.

Also, I think we can safely remove these lines here, since TVM is assumed to be exist in users' environment, IMHO.

Reply:
Thanks! You're right. The import errors were my own fault. I reversed these changes and updated the PR description.